dag/health.yml

```diff
@@ -102,16 +102,23 @@ steps:
   data://grapher/ihme_gbd/2023-06-14/prevalence_dalys_world:
     - data://garden/ihme_gbd/2023-06-14/prevalence_dalys_world
 
-  # IHME GBD Leading cause of child deaths
+  # IHME GBD Leading cause of deaths - to archive
```

Comment:
do you know when is a good time to archive these? Is it after merging?

Reply:
Yeh I think it is typically done in a separate PR